### DISCUSSION TITLE
How do I obtain the node resource usage

### DISCUSSION THREAD

**Discussion by fzwu** [kubernetes-client/csharp#1109] [Q&A]
I use KubernetesClient V8.0.68 and I need to get cluster resource usage such as CPU and memory for cluster monitoring,Includes the Devlopment and Pod status 
How should I call the SDK interface?

**Comment by tg123** [comment]
metrics: you can check https://github.com/kubernetes-sigs/metrics-server and we have `KubernetesMetricsExtensions` in sdk
for deploy and po, you can use get to retrieve the status

**Reply by fzwu** [reply]
Which version of KubernetesClient contains KubernetesMetricsExtensions，V8.0.68 does not include KubernetesMetricsExtensions

**Reply by tg123** [reply]
v8.0+ should have it
